### PR TITLE
display status of meetings

### PR DESCRIPTION
**Pull request by ppelayo1**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
This branch displays the meeting groups by status. Completed groups are displayed in the light gray color, then the pending/active meeting is displayed with a dark blue color, then the remaining meeting groups displayed with a light blue color.

Meetings in progress have their meeting group given an extra line of text "In Progress" under the meeting group title.
Lastly the header of the page indicates in the top right the status of the meeting itself. Either "Not Started" or "In Progress"

- Describe what changes are being made
frontend/src/components/Header/Header.jsx receives as a prop a flag indicating if the meeting is in progress. Code then displayed that status.

frontend/src/components/MeetingView/AgendaView/AgendaGroups.jsx modified to apply different classes to the agenda Groups depending on their status.
Logic checks for "Completed" status for the lightgray color, then for "In Progress"/"Pending" for the dark blue color, then assigns all following "pending" status the light blue color.
**Additional changes** include changing the accordion expansion icon, and displaying a "In Progress" text below the title name if the group is "In Progress"

frontend/src/components/MeetingView/MeetingView.jsx passes the set in progress flag to frontend/src/components/MeetingView/AgendaView/AgendaView.jsx that is used to indicate to Header.jsx the progress status of the meeting

- Describe why these changes are being made
Display agenda groups by their status for Trello 
https://trello.com/c/UTRGYkQy/117-eng-in-progress-status
https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?node-id=195%3A17719

- List the use cases and edge cases relevant to this PR
Desired use would be Completed groups, then pending/in progress groups, then pending groups.
Not following this order will cause adverse results.
A "Pending" then a "In Progress" will result in the "pending" dark blue group above the dark blue "In Progress" group.

Currently the code grabs only the status of the group to indicate if that group is "In Progress", "Completed", or "pending". Code later needs to be implemented to set that group to the correct meeting status. I intend to implement this functionality in another PR solely for that purpose to avoid a overly bloated PR.
- Describe how errors will be handled. How will we know if this code breaks in production

- Describe any external libraries/dependencies added or removed in this PR

- Describe any security risks are there regarding this change

- Describe how you tested these changes
Created different meetings and set different statuses and loaded those pages to ensure the correct output.

- Link to relevant external documentation

--------------------------
### :clipboard: Mandatory Checklist

- [x] Example of a checked item (please remove when creating your Pull Request) 

- [ ] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [ ] Have you discussed these changes with the project leader(s)?
- [ ] Do all variable and function names communicate what they do?
- [ ] Were all the changes commented and / or documented?
- [ ] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [ ] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [X ] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### Before

https://user-images.githubusercontent.com/32974267/149855974-bcd6d77b-0ce2-4d3c-a0c1-96f7d5258d94.mp4

#### After

https://user-images.githubusercontent.com/32974267/149855951-34b189b0-743c-49d1-950b-bbd31016292e.mp4

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

File: frontend/src/components/MeetingView/AgendaView/AgendaGroups.jsx [hunk 1]
```diff
@@ -62,15 +91,88 @@ function AgendaGroups({
...
+  let accordianItemClass = buttonBaseClass;
+  if (active || next) {
+    accordianItemClass += nextClass;
+  } else if (completed) {
+    accordianItemClass += completedClass;
+  }
+  return accordianItemClass;
+};
+
+// outputs the class name for the icon
```
Comment (JMStudiosJoe) [inline frontend/src/components/MeetingView/AgendaView/AgendaGroups.jsx:110]: you give great descriptive cont names, the comments are redundent and not needed but only if you want to add them.

**Review by JMStudiosJoe — COMMENTED** [review]
minor comments

File: frontend/src/components/MeetingView/AgendaView/AgendaGroups.jsx (hunk 1 above)
Comment (ppelayo1) [reply frontend/src/components/MeetingView/AgendaView/AgendaGroups.jsx:110]: Yeah, the comment is unnecessary, do it out of habit sometimes as part of my thought process.

File: frontend/src/components/MeetingView/AgendaView/AgendaGroup.scss [hunk 2]
```diff
@@ -3,51 +3,84 @@
...
-        border-left: 0.5em solid currentColor;
-        transition: 0.1s ease-in transform;
-        position: absolute;
-        left: -15px;
-      }
-
-      &[aria-expanded='true'] .group-title::before,
-      &[aria-selected='true'] .group-title::before {
-          transform: rotate(90deg);
+        .groupStatus{
```
[the remark this answers is not in the document]
Comment (ppelayo1) [reply frontend/src/components/MeetingView/AgendaView/AgendaGroup.scss:23]: Thank you for pointing that out, I don't seem to have lint set up properly for the .scss files or the backend, so going to get that working later.

**Comment by ppelayo1** [comment]
I went and fixed the inconsistency in the stylesheets. Does your ESlint auto catch those issues?
Mine does not.

**Pull request merged**